s3_file_field/views.py

```diff
@@ -121,6 +125,8 @@ def upload_finalize(request: Request) -> HttpResponseBase:
 
     response_serializer = UploadFinalizationResponseSerializer(
         {
+            'finalize_url': finalized_upload.finalize_url,
```

Comment:
I wonder whether we should return this in another endpoint. The downside is that we'd require another full HTTP request-response cycle for every upload flow, but the upside is that we could be certain the client had correctly called finalize (we'd probably need to do an S3 HEAD to confirm this on the server) before we issue them the signed `field_value` to use with the model field.

Reply:
I think the extra request is worth it for the assurance, even if it's more work right now.